app/parsers/bulkrax/csv_parser.rb

```diff
@@ -324,7 +324,7 @@ def retrieve_cloud_files(files)
     def write_files
       require 'open-uri'
       folder_count = 0
-      sorted_entries = sort_entries(importerexporter.entries)
+      sorted_entries = sort_entries(importerexporter.entries.uniq(&:identifier))
```

Comment:
before #597, child collections were exported as a `CsvEntry` class instead of `CsvCollectionEntry`. this means that when that importer/exporter is run again, the old entry still exists, but a new one is created as well. we want to get the newest one.

``` bash
irb(main):148:0> current_record_ids.count
=> 13

irb(main):149:0> Bulkrax::Exporter.find(59).entries.count
=> 14

Bulkrax::Exporter.find(59).entries.uniq(&:identifier).count
=> 13
```